Examples/NoModules/Chapter 02/Ex2_06.cpp

```diff
@@ -0,0 +1,17 @@
+// The width, alignment, fill, and 0 formatting options of std::format() 
```

Comment:
Wrong comment. This is the comment from Ex2_05.
Correct comment:
// Formatting numeric values with std::format()